educator/src/Dscp/Educator/Web/Educator/Types.hs

```diff
@@ -245,10 +322,30 @@ instance Buildable (ForResponseLog [AssignmentEducatorInfo]) where
 instance Buildable (ForResponseLog [SubmissionEducatorInfo]) where
     build = buildListForResponse (take 4)
 
+instance Buildable (ForResponseLog [Certificate]) where
+    build = buildListForResponse (take 4)
+
+instance Buildable (ForResponseLog [a]) =>
+         Buildable (ForResponseLog (Counted a)) where
+    build (ForResponseLog (Counted n mbLs)) =
+        "Counted { n = "+|n|+", items = "+|mbItems mbLs|+" }"
```

Comment:
`buildListForResponse` already displays the number of elements, imo it would be more readable to show only `mbLs` when it is `Just` and only size otherwise